pkg/networkservice/common/updatepath/client.go

```diff
@@ -20,44 +20,35 @@ import (
 	"context"
 
 	"github.com/golang/protobuf/ptypes/empty"
-	"github.com/networkservicemesh/api/pkg/api/networkservice"
 	"google.golang.org/grpc"
 
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
-	"github.com/networkservicemesh/sdk/pkg/tools/token"
 )
 
-type updatePathClient struct {
-	commonUpdatePath
+type idClient struct {
```

Comment:
I'd suggest renaming here to match the new package name.

Reply:
Fixed.